src/app/about/about.component.html

```diff
@@ -0,0 +1,258 @@
+          <p>
+            Dockstore never requires a user to log in to search for workflows and tools. Once found, analyses can be moved from
+            environment-to-environment (e.g. <a target="_blank" rel="noopener" href="https://terra.bio/">Terra</a>,
+            <a target="_blank" rel="noopener" href="https://www.dnastack.com/">DNAstack</a>, and
+            <a target="_blank" rel="noopener" href="https://www.dnanexus.com/">DNAnexus</a>) and yet be guaranteed to run on anything that
+            supports Docker.
+          </p>
+        </div>
+        <div fxLayout="column" fxLayoutAlign="none start">
+          <img src="../../assets/images/about-page/interoperable-icon.svg" alt="icon" class="principles-icon" />
+          <h5>Interoperable</h5>
+          <p>
+            Docker repositories (e.g.
+            <a target="_blank" rel="noopener" href="https://hub.docker.com/" data-cy="Docker-hub-about-link">Docker Hub</a>) and source
+            control repositories (e.g.
+            <a target="_blank" rel="noopener" href="https://github.com/dockstore/dockstore" data-cy="Github-about-link">GitHub</a>) provide
+            much of the needed infrastructure. Contributors are encouraged to provide clear instructions, test data and checker workflows to
+            ensure their software is usable in any environment.
+          </p>
+        </div>
+        <div fxLayout="column" fxLayoutAlign="none start">
+          <img src="../../assets/images/about-page/reusable-icon.svg" alt="icon" class="principles-icon" />
+          <h5>Reusable</h5>
+          <p>
+            One of the ultimate goals of Dockstore is to provide workflows and tools that others can find, reuse and build upon. To ensure
+            this, contributors are encouraged to provide clear documentation and the exact version of the container in their descriptor
+            files.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="register-dockstore pt-5 pb-5">
+      <div fxLayout="row" class="container" fxLayoutAlign="space-between center" fxLayoutGap="3rem">
+        <img fxHide.lt-md src="../../assets/images/about-page/diagram-with-logos.png" alt="diagram" fxFlex.gt-sm="45" class="h-100" />
+        <div fxLayout="column" fxFlex>
+          <h3>Registering with Dockstore</h3>
+          <h5>You can register your workflows and tools on Dockstore in three ways:</h5>
+          <ul class="ordered-list">
+            <li>
+              Dockstore can retrieve your workflow descriptors from GitHub and other source control methods (new with
+              <a href="https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps.html">github apps</a>). You are
+              responsible for ensuring that your descriptors point at valid Docker images
+            </li>
+            <li>
+              You can use our hosted workflows service to store tools and workflows directly on dockstore.org in order to quickly get
```

Comment:
I would drop the "limited audience". That's ultimately the intent, but that feature is not really being used yet.